e2e/testclients/e2etestclient/testsuite_test.go

```diff
@@ -339,13 +339,12 @@ func waitForAcceptedPersonaTag(c *clientutils.NakamaClient) error {
 			return err
 		}
 		status, err := getStatusFromResponse(resp)
-		if err != nil {
-			return fmt.Errorf("unable to get 'status' field from response: %w", err)
-		}
-		if status == "accepted" {
-			break
-		} else if status != "pending" {
-			return fmt.Errorf("bad status %q while waiting for persona tag to be accepted", status)
+		if err == nil {
+			if status == "accepted" {
+				break
+			} else if status != "pending" {
+				return fmt.Errorf("bad status %q while waiting for persona tag to be accepted", status)
+			}
```

Comment:
made this change since the error was saying the tick hadn't processed the persona tag yet. so the first error encountered was stopping the retry mechanism from looping.